packages/fxa-payments-server/src/components/PaymentConfirmation/index.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import MockApp from '../../../.storybook/components/MockApp';
+import PaymentConfirmation from './index';
+import { Customer, Profile, Plan } from '../../store/types';
+
+const userProfile: Profile = {
+  avatar: 'http://placekitten.com/256/256',
+  displayName: 'Foxy77',
```

Comment:
Should these fields be populating?

![image](https://user-images.githubusercontent.com/13018240/75461667-b8baf900-5948-11ea-91ca-7b4c52635f11.png)

Reply:
This is a known issue where variables in Localized elements don't populate in our storybook